debian/postinst

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+resources_dir="${RESOURCES_UNPACK_DIR:-/tmp/ya-installer/}"
```

Comment:
Is there a case where user might want to use different directory?
Do you want to let user to do `export RESOURCES_UNPACK_DIR=$HOME` and then run `rm -r` on it?
If I were given choice to set `RESOURCES_UNPACK_DIR` I would set `$HOME/Download`. I would be pissed if it got deleted.

Reply:
it's good point, it is dangerous.
There is a case, but not in related to debs. I would like to use single script in all places (`ya-installer`) and there you would need to provider directory

Reply:
If this env var is meant only for some internal use then it is fine.